CRM/Volunteer/Hook.php

```diff
@@ -0,0 +1,13 @@
+<?php
+class CRM_Volunteer_Hook
+{
+  static $_nullObject = NULL;
+
+  static function runDefaultSettingsHook(&$defaults) {
```

Comment:
Code doc, please. See [buildForm hook](https://github.com/civicrm/civicrm-core/blob/ef587f9c568bb8e0e310b910c905d61fb507da54/CRM/Utils/Hook.php#L368-L379), for example.